Project.toml

```diff
@@ -15,7 +19,7 @@ VideoIO = "d6d074c3-1acf-5d4c-9a43-ef38773959a2"
 
 [compat]
 FFMPEG = "0.3, 0.4"
```

Comment:
make sure to set upper bounds to the new `[deps]`

Reply:
I reviewed the current package states and felt like I set reasonable boundaries with the versions. I decided upon the boundaries based on knowing when I started working with GTK and the liveviewer. So, at the current boundaries, everything works as expected. If you want me to revise these boundaries, let me know.

Reply:
The boundaries and possibilities are way to specific. I actually wasn't able to `dev Javis`. It works only with Luxor 2.5.1 btw cormullion fixed the issue we had.
My new updated one is:
```
Animations = "0.4"
Cairo = "1.0.4, 1.0.5"
ColorTypes = "0.10"
FFMPEG = "0.3, 0.4"
Gtk = "1.1"
GtkReactive = "~1.0.3"
Images = "0.20, 0.21, 0.22"
LaTeXStrings = "1.1"
LightXML = "0.9"
Luxor = "2"
ProgressMeter = "1"
VideoIO = "0.6, 0.7, 0.8"
julia = "1.4"
```

Reply:
I adopted this scheme. Thanks for the help